packages/moonstone/BodyText/BodyText.js

```diff
@@ -48,18 +64,49 @@ const BodyTextBase = kind({
 		 * @type {Object}
 		 * @public
 		 */
-		css: PropTypes.object
+		css: PropTypes.object,
+
+		/**
+		 * Toggles multi-line (`false`) vs single-line (`true`) behavior. `nowrap` mode
+		 * automatically enables {@link moonstone/Marquee} so long text isn't permanently occluded.
+		 *
+		 * @type {Boolean}
+		 * @default false
+		 * @public
+		 */
+		nowrap: PropTypes.bool
+	},
+
+	defaultProps: {
+		nowrap: false
 	},
 
 	styles: {
 		css: componentCss,
 		publicClassNames: 'bodyText'
 	},
 
-	render: ({css, ...rest}) => {
+	computed: {
+		className: ({nowrap, styler}) => styler.append({nowrap})
```

Comment:
This is technically not necessary at the moment, but I had intended to put a class stub into the stylesheet for future styling. Maybe this can be deleted. Let's see what direction the reviewers think is best.

Reply:
Probably better to remove this and add them later as needed. Otherwise if we are keeping this, might as well add an empty ruleset for `.nowrap`

Reply:
Addressed by adding the nowrap class to the LESS file.